### PR TITLE
Set name of places in English (1/2)

### PR DESCRIPTION
**Pull request by Airon90**
Fixing #286,
Fixed wrong year (next week it should be changed again lol)

### CODE REVIEW

File: res/values/strings.xml [hunk 1]
```diff
@@ -346,8 +346,8 @@ there are any).</string>
     <!-- SBB is german/international, CFF is french and FFS is italian! So this IS translatable. -->
     <string name="np_name_sbb" translatable="true">SBB</string>
     <string name="np_desc_sbb">Switzerland</string>
-    <string name="np_desc_vbl">Luzern</string>
-    <string name="np_desc_zvv">Zurich</string>
+    <string name="np_desc_vbl">Lucerne</string>
+    <string name="np_desc_zvv">Zürich</string>
```
Comment (grote) [inline res/values/strings.xml]: Are you sure Zürich is English?

File: res/values/strings.xml [hunk 2]
```diff
@@ -48,7 +48,7 @@ there are any).</string>
     <string name="tab_devs">Developers</string>
     <string name="tab_about">About</string>
 
-    <string name="about_top">© 2013 &#8211; 2015 Torsten Grote\nFree Software License: GPLv3</string>
+    <string name="about_top">© 2013 &#8211; 2016 Torsten Grote\nFree Software License: GPLv3</string>
```
Comment (grote) [inline res/values/strings.xml]: Please change it to 2017 right away.

File: res/values/strings.xml (hunk 1 above)
Comment (Airon90) [inline res/values/strings.xml]: In the [Wikipedia page](https://en.wikipedia.org/wiki/Z%C3%BCrich) it is used "Zürich" even if it is written that even Zurich is used

File: res/values/strings.xml (hunk 2 above)
Comment (Airon90) [inline res/values/strings.xml]: Done ;)

File: res/values/strings.xml (hunk 1 above)
Comment (grote) [inline res/values/strings.xml]: Thanks! However, I still prefer to leave it as "Zurich" as this is also correct and the main reason: Every time you change a source string *all* translations for this string have to be redone. I don't think this change justifies creating unnecessary work for all translators.

Comment (Airon90) [inline res/values/strings.xml]: Done ;)

**Comment by grote** [comment]
Thanks again! :)

**Pull request merged**